src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -305,6 +314,16 @@ public void setGoalSpeeds(ChassisSpeeds speeds, boolean fieldCentric) {
     }
   }
 
+  public void lockRotationToPosition(Translation2d targetPosition) {
+    lockRotationToPoint = true;
+    lockedTargetPosition = targetPosition;
+    angleController.reset(getRotation().getRadians());
```

Comment:
currently it seems that you are resetting the angleController every cycle. That's not ideal, so could we remove this?

Reply:
this function should be where you set the target position based on the provided option from the command (based on the enum I mention in another comment, and which alliance we are on).